### DISCUSSION TITLE
Stats dashboard

### DISCUSSION THREAD

**Discussion by D10DE** [drakkan/sftpgo#1041] [Ideas]
Hello.
Is there an ability to see connection speed online (live)?
It would be nice to have a live dashboard, which has a table with columns "user", "speed", "estimated time", "file", "progress", etc. Also there might be speed trend, total upload/download, current number of connections.

**Comment by drakkan** [comment]
Take a look at the "Connections" page

![Schermata del 2022-11-03 08-34-43](https://user-images.githubusercontent.com/553263/199668033-a0a1e5e3-8239-48ef-94f7-b4eab11d6252.png)
